app/assets/javascripts/modules/namespaces/pages/index.js

```diff
@@ -30,28 +30,31 @@ $(() => {
     data() {
       return {
         state: NamespacesStore.state,
-        accessibleNamespaces: [],
+        username: window.currentUsername,
+        normalNamespaces: [],
         specialNamespaces: [],
       };
     },
 
     methods: {
       onCreate(namespace) {
-        const currentNamespaces = this.accessibleNamespaces;
+        const currentNamespaces = this.normalNamespaces;
         const namespaces = [
           ...currentNamespaces,
           namespace,
         ];
 
-        set(this, 'accessibleNamespaces', namespaces);
+        set(this, 'normalNamespaces', namespaces);
       },
 
       loadData() {
         NamespacesService.all().then((response) => {
-          const accessibleNamespaces = response.data.accessible.data;
-          const specialNamespaces = response.data.special.data;
+          const namespaces = response.data;
 
-          set(this, 'accessibleNamespaces', accessibleNamespaces);
+          const normalNamespaces = namespaces.filter(n => !n.global && n.name !== this.username);
+          const specialNamespaces = namespaces.filter(n => n.global || n.name === this.username);
```

Comment:
@vitoravelino bear in mind that this doesn't have to be the case. You have to rather trust the user's `namespace_id` attribute.

Reply:
Totally forgot to update that. I saw that you've already fixed it in the model. Thanks for reminding it! :)